tools/modify_master.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+
+import shakedown
+import logging
+
+
+# Methods to modify the envvar settings of a mesos master and restart
+# the master process after they are modified
+
+logger = logging.getLogger(__name__)
+
+
+def set_master_envvar(envvar, value):
+    as_dict = {envvar: value}
+    modify_envvars(lambda envvars: envvars.update(as_dict))
+
+
+def set_master_envvars(new_envvars_as_dict):
+    modify_envvars(lambda envvars: envvars.update(new_envvars_as_dict))
+
+
+def remove_master_envvar(envvar):
+    modify_envvars(lambda envvars: envvars.pop(envvar, None))
+
+
+def modify_envvars(modifier):
+    logger.info("Modifying envvars on master node...")
+    # Get the current set of master envvars
+    success, out = shakedown.run_command_on_master('cat /opt/mesosphere/etc/mesos-master')
+    if success is not True:
+        logger.info('Unable to get current envvars from master: {}'.format(out))
+        raise RuntimeError("Unable to get current envvars")
```

Comment:
Maybe not worth it this time, but ideally some info about the failure (stderr?) should go in here. Probably too much work to figure out the right things to get for now.